lambdas/service/app.py

```diff
@@ -1525,28 +1525,32 @@ def manifest_route(*, fetch: bool, initiate: bool):
     )
 
 
-@manifest_route(fetch=False, initiate=True)
+@manifest_route(fetch=False, method='PUT')
+@manifest_route(fetch=False, method='POST')
 def file_manifest():
     return _file_manifest(fetch=False)
 
 
-@manifest_route(fetch=False, initiate=False)
+@manifest_route(fetch=False, method='GET')
 def file_manifest_with_token(token: str):
     return _file_manifest(fetch=False, token_or_key=token)
 
 
-@manifest_route(fetch=True, initiate=True)
+@manifest_route(fetch=True, method='PUT')
+@manifest_route(fetch=True, method='POST')
 def fetch_file_manifest():
     return _file_manifest(fetch=True)
 
 
-@manifest_route(fetch=True, initiate=False)
+@manifest_route(fetch=True, method='GET')
 def fetch_file_manifest_with_token(token: str):
     return _file_manifest(fetch=True, token_or_key=token)
 
 
 def _file_manifest(fetch: bool, token_or_key: Optional[str] = None):
     request = app.current_request
+    require(request.method != 'POST' or request.raw_body.decode() == '',
```

Comment:
This is the same as

```suggestion
    require(request.method != 'POST' or request.raw_body == b'',
```

Right?